projects/components/src/datagrid/datagrid.component.html

```diff
@@ -37,9 +37,11 @@
     <clr-dg-column *ngIf="shouldDisplayButtonsOnRow()" [ngClass]="'buttons-' + this.getMaxFeaturedButtonsOnRow()">
         Actions
     </clr-dg-column>
+    <!--The clrDgField being passed below is only used for enabling sorting and not filtering. This is because, it was
```

Comment:
Please remove comma after because. Add a space after the opening and closing `<!-- This is a comment -->`

If it needs to be multiline, the opening and closing look better on their own lines

```
<!-- 
  This is just a comment
  that spans multiple lines
-->
```

Lastly, the comment talks about `clrDgField` but I don't see it in the code